modin/core/execution/ray/generic/io/io.py

```diff
@@ -190,6 +190,7 @@ def func(df, **kw):
             keep_partitioning=True,
             lengths=None,
             enumerate_partitions=True,
+            max_retries=0,
```

Comment:
This is a separate change. Please limit this PR to only adding _ability_ to pass extra keywords, and use them separately (you could use that in the `to_csv`-hang-fixing PR after this one is merged).